XenAdmin/Diagnostics/Problems/VMProblem/AutoStartEnabled.cs

```diff
@@ -58,7 +58,7 @@ public override string Description
         protected override AsyncAction CreateAction(out bool cancelled)
         {
             cancelled = false;
-            return new DelegatedAsyncAction(_connection, "", "", null, ActionDelegate(false));
+            return new DelegatedAsyncAction(_connection, string.Format(Messages.DISABLED_AUTOSTART_ON_VM, Helpers.GetName(VM)), "", null, ActionDelegate(false));

```

Comment:
I suggest to have the action's Title saying "Disable VM autostart" and have a separate resource string for the Description, saying "Disabling autostart for 'vm name'..."
Also, please rename the resource strings to something like ACTION_DISABLE_AUTOSTART_ON_VM and ACTION_DISABLING_AUTOSTART_ON_VM, and similar for the enabling action.